### ISSUE TITLE
Develop lotteries theory model

### ISSUE THREAD

**Issue by huntallcott** [huntallcott/Lotteries#5]
The goal of this issue is to develop the lotteries theory model.
Lower-priority than finishing soda taxes, but I'm opening this thread because I had an idea.

Is it possible to develop the following proposition, inspired by @dtaubinsky's logic on the phone over the past few weeks. 
Imagine making some simple parameterizations of curvature of social welfare function, purchases by income, income effects, and whatever else. Assume that gamma is homogeneous by income. Can we solve for the maximum gamma such that lotteries are welfare enhancing, as a function of the other parameters? 
Then we calibrate that maximum gamma with data, and then in the final section we do a bit of work using counterfactual normative consumer or whatever, to shed some light on whether gamma might be that big. Following @dtaubinsky , this could motivate a paper design that de-emphasizes having a solid estimate of gamma.
I started a new theory to-do list google doc, paralleling the to-do lists we have for soda taxes, and put it on dropbox and [here](https://docs.google.com/document/d/1VadRmbkfNfOM04PoK4c7H6UBkRP3e1fGGYMizv0J0_0/edit?usp=sharing).
I am tentatively assigning this to @bblockwood if that's OK.

**Comment by huntallcott** [comment]
Ben posted the below in #4:
@huntallcott I think there's a nice bridge between the logit demand estimation you've set up and a structural weighted expected utility function that we can use for theory. The piece I was most concerned about was the treatment of the utility gain as a function of the jackpot alone, rather than jackpot+wealth, but I think CARA utility (rather than CRRA) actually resolves this problem, and would therefore be a sensible functional form to use for the GMM estimation in your Equation (7) here. At your convenience, skim over Section 1 here and see if you agree?

** 
@bblockwood thanks for doing this!
I have some thoughts. Give a ring?

**Comment by bblockwood** [comment]
@dtaubinsky, @jackblun:

I've been thinking about the normative role of probability weighting, and had the email exchange below with @huntallcott—interested in your thoughts too. 

### Ben question: 
I’ve been thinking about lottery theory this morning, and I just wrote the following paragraph in my brainstorming notes. 

> The total utility derived from lottery consumption can be decomposed into two components: expected utility (from potential the utility gain of winning the prize) and anticipatory utility (derived prior to the lottery's resolution, from scratching the ticket, dreaming about winning, etc.). Although in many settings these components cannot be disentangled, they are conceptually distinct. Imagine a stylized setting in which you have the opportunity to make a purchase on behalf of your future self, after which you'll be given a drug which erases your memory of the transaction. Now consider two purchase decisions: (a) you can purchase a scratch-off ticket for your future self, but it will be scratched by a third party, and your future self will be unaware of the purchase unless the ticket turns out to be a winner; or (b) you can purchase a lottery ticket that is *guaranteed to lose*, but your future self will receive the ticket and will scratch it off, mistakenly believing it could be a winner. The purchase in (a) contains only expected utility, while the purchase in (b) contains only anticipatory utility. Together, a bundle of (a) and (b) is equivalent to one conventional lottery ticket. We take the normative stance that probability weighting in (a) is a normative mistake, whereas probability weighting in (b) is a matter of preference, and is normatively respectable. 

Any thoughts? It’s not exactly a straightforward experiment to implement(!), but at least in principle the distinction seems conceptually well defined, and I find it normatively satisfying.

### HA replied:
this sounds reasonable to me. but i'm also not sure if it maps to the experiment we are running on mturk. imagine we can "debias" people's probability weighting and it changes their lottery ticket demand. that could have acted through changing anticipatory utility or expected utility. 
if your normative stance is inconsistent with what we're doing in the experiment, then either your stance is wrong or the motivation for the experiment is wrong.
it might be worth posting this on github?

### Picking up here:
I’m not sure it’s inconsistent with our experiment—but it does clarify how we might normatively interpret the results. 

1. If we agree that this is the way that rational consumers should be behaving (normatively), then the “counterfactual normative consumer” aspect of our mTurk experiment is still appropriate, provided that consumers who are better at computing EVs are the ones who better apply that skill when introspecting about their preferences over lotteries and evaluating their own expected and anticipatory utilities. 

2. Our MTurk treatment effect does identify bias if it's the expected utility probability weighting which is sensitive to the treatment, rather than anticipatory utility probability weighting. I find that to be the most plausible hypothesis, for a few reasons. 
    1. Keep in mind that anticipatory utility probability weighting arises from real preferences, which are based on the utility that people derive each time they play the lottery (even when they lose). That shouldn’t be something lottery players are easily mistaken about, since they get very frequent reinforcement of it. On the other hand, past experience provides very little evidence with which to internally calibrate the expected utility probability weighting function, so it may be more sensitive to interventions.
    2. In the context of the thought experiment above, the belief that the treatment effect operates on anticipatory utility probability weighting implies that the willingness to pay for a *definitely losing* lottery ticket would go down due to an EV calculation intervention. That doesn’t seem so likely to me. 
    3. In principle this is a testable hypothesis. For example, we could isolate expected utility probability weighting by eliciting WTP (or probability equivalents) for giving some third party participant a lottery ticket without their knowledge. If the treatment effect is operating only (or mostly) on anticipatory utility probability weighting, then it should have no (or little) effect in such a domain, where only expected utility is relevant. I conjecture the treatment would be just as strong there though.
3. Suppose (2) is wrong, and the treatment effect *does* operate on anticipatory probability weighting. I don’t think that means this normative stance is wrong—rather, it formalizes a critique that people might raise about normatively interpreting our treatment effect as a debiasing intervention. Specifically, this amounts to asserting that the intervention endogenously reduces lottery demand because it’s less fun to play the lottery after one reflects upon the EV of playing, even though it was perfectly rational to play absent the intervention. 
4. Finally, I think this setup can help support the counterfactual normative consumer approach in this setting, since it clarifies why even rational consumers might want to purchase (negative EV) lottery tickets in reality. Put differently, without this allowance for normatively respectable anticipatory utility probability weighting, one could reasonably ask why we should interpret the treated individuals (or the normative benchmark consumers) as unbiased, rather than only partially debiased along a scale that ends with full EV maximization and zero lottery purchases. This setup has an easy response: even normative benchmark consumers are allowed to get anticipatory utility from lottery plays, so we can’t infer that more EV-maximizing behavior is more rational without bound.

**Comment by dtaubinsky** [comment]
@bblockwood: brilliant!! I agree that our EV treatment should only operate on choices in scenario (a).

@bblockwood, do you have thoughts on a variation of this that could be implemented? 

Here's my implementation idea:  A relatively faithful version of the thought experiment is to have participants make gifts to other participants. This would have two stages:

1) Isolate the participants who are somewhat prosocial. That is, ones who are willing to give up some of their own money to increase rewards of others (say, in a modified dictator game). This step is important b/c if you don't care about others at all, every choice is a weakly dominant choice, and so the choices are totally uninformative.

2) Elicit the certainty equivalent (e.g., the minimum certain $X to the other participant that is preferred over the lottery product) for the lottery products in (a) and (b).

This is pretty easy to do on MTurk, which is great!

**Comment by dtaubinsky** [comment]
Sorry, two more notes on that:

1. In the (a) version, it is important that the decider never learns the outcome in both the positive and negative case. This is to prevent the decider from getting anticipatory utility from having the outcome of someone else revealed.

2. It'll be useful to elicit a certainty equivalent for just a standard lottery ticket as well.

**Comment by dtaubinsky** [comment]
Woops, didn't carefully read through 2iii, so yes, we're on the same page (sorry didn't mean to take credit for that).

I think that (a) and standard lottery ticket is actually better because (b) might generate a demand effect by virtue of being weird, as you pointed out. 

I doubt that showing the EV treatment operates equally on the conventional and the (a) version would help the case that much. But I think the combination of i) a correlation between EV computing ability and lottery demand ii) showing that the EV treatment reduces lottery demand and iii) showing that people are still risk-seeking when anticipatory utility is eliminated would be a pretty convincing set of evidence that probability weighting is at least partly a mistake.

**Comment by bblockwood** [comment]
Not at all! Glad we're on the same page. I agree with you on all counts. 

There's also the question of levels and changes. I do think it would provide some support for this model if people exhibit a lower *level* of certainty equivalent for (a) than for conventional (since that supports the existence of anticipatory utility) yet a similar treatment effect for each (since that supports the effect operating on expected utility p weighting, rather than anticipatory utility p weighting.)

We should keep thinking about whether there's a more natural way to elicit WTP for anticipatory utility without demand effects though. One proto-idea: respondent x can pay some amount to ensure that a lottery ticket given to person y is "real", rather than a guaranteed loser. (y knows all this.) What is x's WTP to make the ticket real rather than fake? Is it smaller than x's WTP to give y a real ticket rather than nothing? The spread, if any, is the WTP for y's anticipatory utility. (Not perfect, but something along these lines might be informative—and it would be nice to check whether the EV treatment effect is small/nonexistent then.)

**Comment by dtaubinsky** [comment]
For a direct test of anticipatory utility, what about just eliciting WTP to turn (a) into a standard lottery ticket? That still generates a demand effect, but it's as direct as possible. My fear is that the WTP for turning (a) into a standard ticket may bend up being larger than the difference in WTP for the standard ticket versus (a).

**Comment by bblockwood** [comment]
Yep, I like this idea. Perhaps there's a way to do this elicitation interface that enforces consistency, e.g., same screen, elicit certainty equivalent for (a) and additional amount for converting to standard ticket, with total CE for standard ticket displayed alongside.

**Comment by dtaubinsky** [comment]
Yes, but that we'd still probably sleep easier if that kind of consistency requirement didn't change how much individuals value the standard ticket....

**Comment by bblockwood** [comment]
Agreed. We could have a treatment in which they first answer the standard ticket, then reveals a follow up which asks for how much of that total they’d be willing to pay if the recipient weren’t going to be informed about the ticket.

**Comment by dtaubinsky** [comment]
That's good. Though we'd still want to make sure that this doesn't affect how people value (a) without the preceeding valuation.

**Comment by dtaubinsky** [comment]
Sorry, that was unclear. I'm saying that how much people would value (a) if that was the first valuation they did might be different than the WTP elicited in this sequential set-up.

**Comment by jackblun** [comment]
I very much like the decomposition of the ticket utility into an anticipatory element and a probability weighting element and the follow-up discussion above. 

I'm just wondering how confident we would be that the method proposed for eliciting (a) would truly eliminate anticipatory utility on the part of the decider? If we think that the anticipatory utility is partly 'daydreaming' about possible states of the world, isn't it possible that the decider still receives some anticipatory utility themselves even when they will not learn how much the receiver wins?

**Comment by dtaubinsky** [comment]
@jackblun That's an excellent point. My view is that no approach by itself is convincing, but that together the three will be convincing. One other thing to consider is that we can should compare estimates of probability weighting for self versus others. To the extent that we expect day-dream utility to have a more powerful impact on own probability weights than others, showing that the difference between own and others probability weights is non-existent or small (which is my prior) would be a useful demonstration.

One way to do this paper is to answer the question "Why do people play the lotto and what are the welfare implications?" So we begin by providing interesting new facts about that support the idea that people are making mistakes. But then we show theoretically that this doesn't necessarily imply lottos are bad for welfare, and provide some calibrations.

**Comment by jackblun** [comment]
@dtaubinsky I see - yes that sounds like a good plan.

**Comment by bblockwood** [comment]
Hi @jackblun, @huntallcott, and @dtaubinsky: 

I have been working on lottery theory for the past couple days, and wanted to share what I've learned. See the current writeup [here](https://github.com/huntallcott/Lotteries/blob/master/Output/Lyx/Paper.pdf). Section I.A goes through the model setup we've discussed before, where Equation (1) is the starting point for the demand system @jackblun and @huntallcott will be estimating. 

* The key new parts are Sections I.B — I.E. It occurred to me that we should be able to impose some real structure on the appropriate weighting function by exploring what they would imply about the profit-maximizing lottery. Although we may not want to assume current lotteries are exactly profit maximizing, it would be weird if they didn't cater to people's desires at least approximately. So Sections I.B and I.C derive conditions for the profit maximizing lottery, and Section I.D goes through a numerical example. 
* It turns out that standard probability weighting functions (like the one currently in the demand estimation notes) are *not* able to match lottery demand. (They don't result in optimal jackpots in the right range, and they result in willingness to pay for lottery tickets that is much too high.) 
* However, if we use a probability weighting function only for the anticipatory component of the weighting function, rather than the whole thing, we get pretty reasonable results—especially if we use the weighting function from Prelec (1998 - ECMA). 
* Finally, Section I.E derives expressions for the optimal implicit tax, which of course depends on bias. That expression (with a sufficiently fully estimated system) would pin down the optimal implicit tax rate, as well as the optimal skewness of the lottery. 

My next steps are to understand what predictions this functional form has for things like income effects and the preference for smaller prizes.

**Comment by huntallcott** [comment]
@bblockwood this is truly tremendous work. Thanks for moving this ball forward.
A couple of things to at least consider:
1. I have reservations about taking the Section 1.4 simulations seriously -- you say that standard probability weighting functions can't match demand, but you use alternative weighting specifications, you can. But couldn't we just add some other degree of freedom and also match demand? 
2. Love the optimal implicit tax formulas. One comment: another important research question we've discussed is just, "are current lotteries welfare-improving?" I think this just requires writing a social welfare function and then calibrating it with a demand model, but just to flag that eventually will probably want that equation somewhere, right?

Thanks again for this great work.

**Comment by bblockwood** [comment]
Thanks for these comments, @huntallcott.
1. I don't know if the simulations in 1.4 have exactly the right form, but there is one way in which I think they should be taken quite seriously: they show us *why* the conventional KT and Prelec functions don't match well. Specifically, any weighting function which matches observed lottery demand (assuming lotteries are anything close to profit maximizing) must increase very quickly starting from 0, but then "level out" at a very small pi. Although the KT and Prelec functions have that general shape, they don't "increase then level off" nearly fast enough. You're exactly right that we should be able to do better by adding a parameter -- but then the question is how that parameter should enter. Both the KT and Prelec functional forms have versions that allow for a second parameter (equations 3.2 and 3.6 of Prelec 1998), but in both cases, that secondary parameter doesn’t help us match this feature of lotteries. (Those secondary parameters adjust the point at which the weighting function intersects the 45 degree line.) So to deal with this, the goal of Section 1.4 is to propose a different way to add a second parameter, κ, motivated by the psychological model underlying our weighting function (which decomposes into expected and anticipatory utility). That turns out to work quite well, and it also allows for a straightforward connection between our demand model and our welfare function. Since w(π)=π+f(π), where f() is an S-shaped function, the normative question is then how much of f() can be attributed to anticipatory utility vs. misunderstanding π in expected utility. 
2. I agree this welfare question is central, and I think the utility function proposed in 1.4 lets us address this in an transparent way. Remember that the difference between actual probability and our probability weighting function estimated from lottery demand, w(π)–π=f(π), can be attributed either to welfare-relevant anticipatory utility or to welfare-irrelevant misunderstandings of π. Say we attribute a constant share ɸ to the former. Then ɸ represents a nice parameterization from 0 to 1 of lottery demand coming from a bias vs. a preference. Then there are many interesting questions we can ask, like “what is the threshold ɸ such that existing lotteries are better than nothing” and “how does the optimal lottery change as ɸ varies from 0 to 1”, etc.

**Comment by huntallcott** [comment]
@bblockwood two questions on the model in the slides:
1. For the optimal tax stuff, if we are just using the predicted normative consumer to calculate bias and thinking of "lotteries" as a homogeneous good with one price elasticity for the formulas, do we need all the structure on what a lottery is?
I think we will eventually need it to map to the logit model, but I think it's the case that we don't need it for the optimal implicit tax formulas, right? I just wanted to confirm what objects we need when in the paper.
2. Right now the only bias in your model is because people misinterpret probabilities. But in the surveys, we are looking at gathering info on misunderstanding the happiness gain, as well as "I should play the lottery less," which is present bias. Should we have a generic money-metric bias?

**Comment by bblockwood** [comment]
Thanks @huntallcott. 

1. You are right, we don't need the specifics of the utility function to calibrate the sufficient statistics formula. The reason I nevertheless led with the utility function is to lay out how we're conceptually allowing for lottery demand to be either a bias or normatively justified. Specifically, I wanted to lay out why it might be normatively justified to buy lottery tickets even if one is not risk-seeking (i.e., even if u(.) is globally concave) since that's something that seems confused in the existing literature. I'm open to switching up the order, but I was hoping to front-run a bunch of questions which I think would pop up otherwise about whether lottery demand must be a bias or a preference. 
2. The slides are still evolving, but the [current version](https://github.com/huntallcott/Lotteries/blob/master/paper/output/Lotteries_Slides.pdf) does have misperceived utility gains ($\Delta \tilde U$ on pp. 10-11). That said, it doesn't have present bias — and I agree it should have that too in the structural calibration. As for whether we can do a generic money-metric bias, let me think a little more about this. I think it may be hard to do nonparametrically, since we don't a price elasticity of demand to rescale from quantity-effect of bias to a price difference (as we did with soda taxes).

**Comment by huntallcott** [comment]
1. I think you can already justify lottery ticket purchases in the rational model with positive e (execution utility). So more precisely I think the mathematical benefit here is to explain why something that looks like probability weighting need not be a bias. In my mind the biggest benefit is that it is a "more complete model" of lottery demand.
2. Hmm good point. So for the optimal implicit tax formula, given that we don't have an upfront price elasticity, I hope that this is something you can write in terms of the quantity effect of bias and the elasticity of purchases wrt winnings?

**Comment by bblockwood** [comment]
1. Agreed that execution utility can also help explain rational demand for lotteries, but then don't we still need to lay out the individual utility function so that people can understand what we mean by execution utility? And secondarily, execution utility would not explain systematic variation in demand with jackpot variations, right? I think we want to emphasize that such variation can be normatively respected in our conception.
2. I think we can, provided we place some additional functional structure on the problem — that's what I've tried to do with the current expression for gamma in the slides: 
    <img width="888" alt="image" src="https://user-images.githubusercontent.com/956485/61411348-30557f80-a8b4-11e9-8cc0-838598d16e61.png">
    I think this is related to your [question](https://github.com/huntallcott/Lotteries/issues/33#issuecomment-512269794) from #33 — we can write the FOC for the optimal prize tax in terms of reduced form statistics, but in order to actually estimate that bias statistic, I think we need an estimate of the probability weighting function (if we don't have price variation in lottery tickets).

**Comment by huntallcott** [comment]
2. OK so this is really useful discussion. If we really need a good estimate of the probability weighting function, does that suggest we may want more than just the two MPLs we have right now? Seems like it would be fantastic to nail this down.

**Comment by bblockwood** [comment]
Yes I agree, and that is why I think it would be helpful to ask a range of MPL questions in the auxiliary experiment. 

That said, this discussion has given me a new thought — I am wondering if we can characterize the reduced form optimal tax not in money metric relative to the up-front price, but money metric in terms of reduced winnings. We'll have an elasticity for that, so we can do the conversion from quantity to money metric. On the other hand it's more complicated as that money metric is itself inside something that is subject to bias. I'm working through this this afternoon to see if we can get anything useful there.

**Comment by huntallcott** [comment]
@bblockwood fantastic and thank you and I'm excited to see what your large brain finds here!

**Comment by bblockwood** [comment]
@huntallcott I think I have made some progress on this front — a revised sufficient statistics expression for the optimal tax is in the slides (p. 9), but the welfare weights are a bit different than in RST — rather than representing the marginal social value of _consumption_ for each type, they represent the marginal social value of expected prize winnings. I have some thoughts on how we could convert, but that likely does require more structural assumptions. Perhaps we can discuss in person when I see you.

**Comment by bblockwood** [comment]
@huntallcott: quick update, I have been working on this for a lot of today, and I do not yet see a way for us to fully characterize the reduced form tax without calibrating a weighting function. I'm continuing to explore this, but at this point I'm thinking we will need to be somewhat structural (like the [post above](https://github.com/huntallcott/Lotteries/issues/5#issuecomment-512567678)). To be continued..

**Comment by huntallcott** [comment]
@bblockwood is this true even when we have elasticity of demand wrt all prizes, i.e. the parimutuel case?

**Comment by bblockwood** [comment]
Yes, this is the case even when a “lottery” is just a single {prize,probability} combination. The key issue is that we only measure one elasticity of demand (expenditure with respect to prizes) but we have multiple dimensions of choices over which to be biased (lottery jackpot size, and how many tickets to buy). That’s fine if we impose some structure, but it’s a problem if we try to be completely structure-free like in RST.

**Comment by huntallcott** [comment]
@bblockwood I see. You're the expert, but is there some reasonable way to impose structure on the bias? For example, what if the bias takes the form of increasing the "rational" purchases of each type of lottery by some constant? Or increasing the perceived marginal utility of all possible payouts by some constant?

**Comment by bblockwood** [comment]
Thanks @huntallcott -- yes! I didn't mean to be overly negative -- I think it's perfectly feasible to characterize the optimal tax with additional structure. I'm just saying I don't think we can do it without any such structure, which wasn't initially clear to me. I continue to think the best structure to impose is to approximate a weighting function with MPLs and to assume CRRA utility over consumption. But there are other structural assumptions one could make instead.

**Comment by bblockwood** [comment]
@huntallcott an additional update: I continue to think that we cannot get a reduced form optimal tax calculation without any additional structure. However I think I see a way to get a calculation without eliciting a full weighting function. The approach is described in Section II.A [here](https://github.com/huntallcott/Lotteries/blob/master/paper/output/Paper.pdf). 

The upshot is that if we added two questions to our primary survey, we could compute the reduced form optimal tax, without relying on weighting function information from the auxiliary survey. The additional questions are MPLs, in which we elicity a WTP for two hypothetical lottery tickets, one of which has winning probability π and net prize J, the other of which has the same winning probability and net prize J*(1+k), where k is something like 0.5. 

One concern is that this effectively involves measuring a second moment using the predicted normative consumer approach — we need to measure the difference in WTP between these two MPLs for a _normative_ consumer, which may be difficult if there is a lot of noise in these WTP elicitations.

**Comment by huntallcott** [comment]
@bblockwood this is just tremendous. I had some thoughts and questions on this tremendous work.

1. Great abstract. :-) But seriously this is such an exciting research question. 

2. Minor: it took me awhile to figure out what J was, because usually we think a jackpot as being *after* the lottery agency takes its take but *before* income taxes, and I didn't realize that it was before both. I would clarify the last sentence, which is hinting at that.

3. How do I think about the interaction between this and the income tax? So imagine you wanted to actually use this formula to compute the actual optimal implicit tax for state X, either holding constant their income tax system or allowing that to be endogenous. Is that going to be possible here?

4. Bottom of page 3: aren’t s’.J and s’.t related, so could you simplify this expression? (Note I'm often using periods instead of underscores because markdown doesn't like underscores.)

5. Equation 5: I understand the equation for \rho, but where do a’ and \Delta V’ come from to actually calibrate the formula?

6. Most substantive question: it seems like the biggest problem with this model is that it only has one jackpot. This seems great for building intuition and providing an example formula. But it doesn't seem useful for actually calculating an optimal tax, for two reasons. (i) it's prima facia unrealistic, and (ii) all the statistics like \bar(s), \rho, etc. are going to come from data on purchases of actual lottery tickets with a variety of prize levels, so we have internally inconsistent statistics going into the formula. Is there any way you can extend this already Herculean effort to allow a variety of prizes? 

7. Dumb question about this key statistic v'.J(\theta). Roy's identity allows one to translate derivative of objective function into Marshallian demand. v'.J kind of seems like a derivative of the objective function. Are you sure there's no way of doing something like that? Feel free to ignore.

Presuming that (7) is moot, here are some comments on estimating (1+a/ pi)* Delta V:
8. On estimating  is there possibly a way to do this where we combine the two types of biases (in curvature vs. probability) instead of needing to get them separately? It seems like this is taking the data really seriously. An option I like better is what I think you suggested in your post (which I interpreted differently than the pdf): what if we get W2-W1 for all survey participants, regress that on the vector of bias proxies, and predict what it W2-W1 would be for the consumer with the "normative" values of the bias proxies? 
9. What happens when you apply my comment 6 to the hypothetical lottery tickets you are proposing? Imagine that you are able to extend the framework to have more than just one jackpot. Given that our s and rho statistics are for "all lotteries" or "the average lottery," I think that for internal consistency we want those hypothetical lottery tickets to be more representative of the average lottery, so we might have pair of pretty standard scratchoffs scaled up by factor K, and another pair of pretty standard Megabucks tickets scaled up by factor K.
